kubernetes/resource_kubernetes_job_v1.go

```diff
@@ -48,6 +51,78 @@ func resourceKubernetesJobV1() *schema.Resource {
 	}
 }
 
+func resourceKubernetesJobV1CustomizeDiff(ctx context.Context, d *schema.ResourceDiff, meta interface{}) error {
+	if d.Id() == "" {
+		log.Printf("[DEBUG] Resource ID is empty, resource not created yet.")
+		return nil
+	}
+
+	// Retrieve old and new TTL values as strings
+	oldTTLRaw, newTTLRaw := d.GetChange("spec.0.ttl_seconds_after_finished")
+
+	var oldTTLStr, newTTLStr string
+
+	if oldTTLRaw != nil {
+		oldTTLStr, _ = oldTTLRaw.(string)
```

Comment:
You can just omit the `, _` here if you're not going to check the type assertion.

Reply:
,_ is not omitted, thanks for the feedback!